arviz/data/io_cmdstan.py

```diff
@@ -212,6 +212,8 @@ def sample_stats_to_xarray(self):
                 key_, *end = key.split(".")
```

Comment:
Also, according to this [comment](https://github.com/arviz-devs/arviz/pull/1063#issuecomment-589771200), stan variables don't need to be splitted and joined as none of them '.' seperated strings in their names.